backend/lambdas/api/repo/repo/services/ado.py

```diff
@@ -172,7 +174,7 @@ def _queue_repo(
     # Queue the repo
     scan_id = AWSConnect().queue_repo_for_scan(
         name=org_repo,
-        repo_url=repo["remoteUrl"],
+        repo_url=repo["webUrl"],
```

Comment:
`remoteUrl` prepends the ADO organization where we want the API Key to go (ex: `https://<org>@<vcs>`). We change to the `webUrl` which does not.